### PR TITLE
DIS-155 Integration of Selected OT Tool

### PR DESCRIPTION
**Pull request by hhsons**
### What

Integration of Selected OT Tool for Frontend Cookie Controller. This change allows us to see and analyse requests to/from the service.

This PR does not address Kafka instrumentation, as this is being handled in the Kafka service.

### How to review

Run this service, and the services required to analyse OT data. 
Open `http://localhost:24100/` in your browser. 
View OT through Jaeger UI and see that the call to this service was acknowledged by OT.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -11,13 +12,17 @@ import (
 	"github.com/ONSdigital/dp-frontend-cookie-controller/config"
 	"github.com/ONSdigital/dp-frontend-cookie-controller/routes"
 	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dphttp "github.com/ONSdigital/dp-net/http"
```
Comment (charlescolbourn) [inline main.go:15]: this looks like the import is duplicated?

File: main.go [hunk 2]
```diff
@@ -11,13 +12,16 @@ import (
...
+	dphttp "github.com/ONSdigital/dp-net/v2/http"
+	dpotelgo "github.com/ONSdigital/dp-otel-go"
 	render "github.com/ONSdigital/dp-renderer/v2"
 	"github.com/ONSdigital/dp-renderer/v2/middleware/renderror"
 	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/gorilla/mux"
 	"github.com/justinas/alice"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 
-	dpnethttp "github.com/ONSdigital/dp-net/v2/http"
```
Comment (charlescolbourn) [inline main.go:20]: This seems to be removing an import that's added a few lines above with a different alias. The net effect is just to rename the alias, which feels like an unnecessary change

File: main.go [hunk 3]
```diff
@@ -93,7 +116,7 @@ func run(ctx context.Context) error {
 	}
 }
 
-func gracefulShutdown(cfg *config.Config, s *dpnethttp.Server, hc health.HealthCheck) error {
```
Comment (charlescolbourn) [inline main.go:96]: see above.

File: main.go (hunk 2 above)
Comment (hhsons) [reply main.go:20]: Renamed back to the original alias

File: main.go (hunk 3 above)
Comment (hhsons) [reply main.go]: Renamed back to the original alias

File: README.md [hunk 4]
```diff
@@ -16,6 +16,8 @@ If on go 1.12 then ensure the project either resides outside of your `GOPATH` or
 | GRACEFUL_SHUTDOWN_TIMEOUT    | GracefulShutdownTimeout    | Time to wait during graceful shutdown                                                  | 5 seconds               |
 | HEALTHCHECK_INTERVAL         | HealthCheckInterval        | Interval between health checks                                                         | 30 seconds              |
 | HEALTHCHECK_CRITICAL_TIMEOUT | HealthCheckCriticalTimeout | Amount of time to pass since last healthy health check to be deemed a critical failure | 90 seconds              |
+| OTEL_EXPORTER_OTLP_ENDPOINT         | localhost:4317                         | Endpoint for OpenTelemetry service
+| OTEL_SERVICE_NAME                   | dp-frontend-cookie-controller          | Label of service for OpenTelemetry service
```
Comment (franmoore05) [inline README.md:20]: Could the | characters be brought in line with the rows above please?

Comment (hhsons) [reply README.md]: Aligned the rows. I was also using the default values in the wrong place so have corrected that too.

**Review by franmoore05 — APPROVED** [review]
